client/asset/dcr/dcr.go

```diff
@@ -759,6 +785,31 @@ func (dcr *ExchangeWallet) feeRate(confTarget uint64) (uint64, error) {
 	return 1 + uint64(atomsPerKB)/1000, nil // dcrPerKB * 1e8 / 1e3
 }
 
+// externalFeeEstimator gets the fee rate from the external API
+func externalFeeEstimator(ctx context.Context, nb uint64) (float64, error) {
+	url := externalApiUrl + "/utils/estimatefee?nbBlocks=" + strconv.FormatUint(nb, 10)
+	r, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
```

Comment:
Still need a timeout set.  The `dcr.ctx` is just to cancel requests on shutdown.  Can either wrap this `ctx` using `context.WithTimeout`, or you can change the client definition to `client http.Client{Timeout: t}`.  I'm not sure if there's a practical difference TBH.